assets/js/more-link.js

```diff
@@ -0,0 +1,8 @@
+( function() {
+	var links = document.querySelectorAll('.more');
+
+	links.forEach( link => {
+		var post = link.previousElementSibling;
+		post.appendChild(link);
+	} );
```

Comment:
We don't have babel and we need to support IE11

```suggestion
	links.forEach( function( link ) {
		var post = link.previousElementSibling;
		post.appendChild(link);
	} );
```